apps/usecase/get_order.py

```
@@ -0,0 +1,24 @@
+    def execute(self, query: GetOrderQuery) -> tuple[dict, int]:
+        try:
+            doc = self.order_read_repo.get_by_id(query.order_id)
+            if not doc:
+                return {
+                    "error": "order_not_found",
+                    "message": f"La orden {query.order_id} no fue encontrada"
+                }, 404
+
+            return doc, 200
+
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)